libs/SalesforceSDK/res/layout/sf__login.xml

```diff
@@ -12,37 +12,41 @@
         android:scrollbars="none"
         android:windowDrawsSystemBarBackgrounds="true"/>
 
-    <Button android:id="@+id/sf__idp_login_button"
-        android:text="@string/sf__launch_idp"
-        android:visibility="gone"
-        android:onClick="onIDPLoginClick"
-        android:layout_width="match_parent"
-        android:layout_height="50dp"
+    <LinearLayout
```

Comment:
Wrapping both buttons ("Biometric Login" and "Login with IDP") in LinearLayout so that they don't overlap when both shown.